src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -172,7 +212,36 @@ function activateInjectedAssets() {
       );
     })
     .then(headerFooterData => {
-      mountReactComponents(headerFooterData, createCommonStore());
+      teamsitesSetup();
+
+      const headerContainer = document.createElement('div');
+      headerContainer.classList.add('ts-header-container');
+
+      const footerContainer = document.createElement('div');
+      footerContainer.classList.add('ts-footer-container');
+
+      renderHeader(headerFooterData.megaMenuData, headerContainer);
+      renderFooter(headerFooterData.footerData, footerContainer);
+
+      window.addEventListener(
```

Comment:
This section keeps the responsive styling and VCL banner/modal functionality working when the window is resized. Since the Teamsites header/footer is largely not React code anymore, we have to manually plug in toggling between the mobile and desktop code when the window is resized (and add debouncing of course).